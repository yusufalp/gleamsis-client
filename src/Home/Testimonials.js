import React from 'react';

class Testimonials extends React.Component {
  render() {
    let testimonials = [
      {
        quote: 'Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.',
        author: '—Aldous Huxley,',
        cite: 'Brave New World'
      },
    ]
    return (
      <div>
        {testimonials.map((testimonial, i) =>
          <blockquote key={i} cite="https://www.huxley.net/bnw/four.html">
            <p className='quote'>{testimonial.quote}</p>
            <p>{testimonial.author} <cite>{testimonial.cite}</cite></p>
          </blockquote>
        )}
      </div>
    )
  }
}

export default Testimonials;