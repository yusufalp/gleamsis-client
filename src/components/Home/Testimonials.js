import React from 'react';

class Testimonials extends React.Component {
  render() {
    let testimonials = [
      {
        quote: 'GleamSIS has saved us more than half of the time we used to spend organizing data about our students, parents and teachers. Now with a few clicks we can access it right away. User friendly, practical and just exactly what we need.',
        author: 'â€”Sevde Meva Alp,',
        cite: 'Principal at That School'
      },
    ];
    return (
      <div>
        {testimonials.map((testimonial, i) =>
          <blockquote key={i} cite='https://www.huxley.net/bnw/four.html'>
            <p className='quote'>{testimonial.quote}</p>
            <p>{testimonial.author} <cite>{testimonial.cite}</cite></p>
          </blockquote>
        )}
      </div>
    );
  }
}

export default Testimonials;