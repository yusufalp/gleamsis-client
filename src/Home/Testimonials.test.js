import React from 'react';
import ReactDOM from 'react-dom';
import Testimonials from './Testimonials';
import renderer from 'react-test-renderer';

describe('Testimonials component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Testimonials />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create( <Testimonials />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})