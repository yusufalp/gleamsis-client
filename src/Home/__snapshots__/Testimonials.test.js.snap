// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonials component renders the UI as expected 1`] = `
<div>
  <blockquote
    cite="https://www.huxley.net/bnw/four.html"
  >
    <p
      className="quote"
    >
      GleamSIS has saved us more than half of the time we used to spend organizing data about our students, parents and teachers. Now with a few clicks we can access it right away. User friendly, practical and just exactly what we need.
    </p>
    <p>
      â€”Sevde Meva Alp,
       
      <cite>
        Principal at That School
      </cite>
    </p>
  </blockquote>
</div>
`;
